test/hello-query.ts

```diff
@@ -26,4 +36,55 @@ describe('hello query test', function () {
     const content = response.data.data.hello[0].content
     expect(content).to.be.equal('Hello world!')
   })
+
+  const test_input = {
+    data: {
+      name: 'test_name',
+      email: 'test_email1',
+      password: 'test_password1',
+      birthDate: '01-01-1900',
+    },
+  }
+
+  it('Should create a new user\n', async function () {
+    const response = await axios.post('http://localhost:4000', {
+      query: `
+        mutation CreateUser($data: UserInput!) {
+          createUser(data: $data) {
+            birthDate
+            email
+            id
+            name
+          }
+        }
+      `,
+      variables: {
+        data: {
+          name: test_input.data.name,
+          email: test_input.data.email,
+          password: test_input.data.password,
+          birthDate: test_input.data.birthDate,
+        },
+      },
+    })
+
+    const user = await prisma.user.findUnique({
+      where: {
+        email: test_input.data.email,
+      },
+    })
+    createdUserId = response.data.data.createUser.id
+    expect(response.data.data.createUser).to.deep.include({
+      name: test_input.data.name,
+      email: test_input.data.email,
+      birthDate: test_input.data.birthDate,
+    })
```

Comment:
Principalmente pra responser, o `to.be.deep.eq` é mais completo. Dá pra vc "separar" o id com spread:

```ts
const { id, ...userFields } = response.data.data.createUser;
expect(userFields).to.deep.eq({
      name: test_input.data.name,
      email: test_input.data.email,
      birthDate: test_input.data.birthDate,
})
```